km_api/know_me/models.py

```diff
@@ -50,6 +50,55 @@ def get_km_user_image_upload_path(km_user, imagename):
             id=km_user.id)
 
 
+class EmergencyItem(mixins.IsAuthenticatedMixin, models.Model):
+    """
+    An emergency item holds emergency information for a KMUser.
+
+    Attributes:
+        description (optional, str):
+            The description for the KMUser.
+        media_resource (optional):
+            A ``Media Resource`` associated with the emergency item.
+        km_user:
+            The KMUser the item is connected with.
+        name:
+            The name of the item.
+    """
+    description = models.TextField(
+        blank=True,
+        default='',
+        verbose_name=_('description'))
+    media_resource = models.ForeignKey(
+        'know_me.MediaResource',
+        blank=True,
+        null=True,
+        related_name='emergency_items',
+        related_query_name='emergency_item',
+        verbose_name=_('media resource'))
+    km_user = models.ForeignKey(
+        'know_me.KMUser',
+        related_name='items',
+        related_query_name='item',
```

Comment:
I think it would be more useful if we had the related name be something like `emergency_items`. That way we would access the emergency items with `km_user_instance.emergency_items`, which makes it more clear what type of items they are.

I think `related_name=emergency_items` and `related_query_name=emergency_item` would take care of that.